plugins/nginx.yaml

```diff
@@ -1,36 +1,11 @@
-version: 0.0.6
+version: 0.0.7
 title: Nginx
 description: Log parser for Nginx
 supported_platforms: 
   - linux
   - windows
   - macos
-  - kubernetes
```

Comment:
Talked to Joe about this, and apparently we decided that both this plugin and the ingress one should be supported on kubernetes. I'm not sure if this means that we need to persist this entire functionality given that information, but probably something we should discuss furhter.

Reply:
You are correct I had removed kubernetes options completely. I have reverted changes and only removed the ingress portion of plugin.